### ISSUE TITLE
Assassin job change lock up.

### ISSUE THREAD

**Issue by LiamKarlMitchell** [rathena/rathena#2070]
* **rAthena Hash**: 948e2bdc0a70174c76fd303dfa2cee376c2d639d

* **Client Date**:  2015-09-16a

* **Server Mode**:  Pre-Renewal (Probably both tho)

* **Description of Issue**: 
  * Result: During part of the job change you are tele-ported into a room and have to walk to next warp. However it locks up seemingly until NPC timeout due to being out of range of npc.
  * Expected Result: NPC prompt should end and let you walk.
  * How to Reproduce: Do assassin job change.

I get the following warning in console.
```
[Warning]: npc_script_cont: failed npc_checknear test.
```

It is in this room.
![image](https://cloud.githubusercontent.com/assets/1968109/25074482/b06cfe74-234f-11e7-8cdd-de0d63231d81.png)

I'll try to get a /where

**Comment by Jeybla** [comment]
Can you provide more explicit informations about where the error happens? The screen could be anywhere in the assassin job quest. After or before which part of the quest does this error happen? Thanks in advance!

**Comment by Jareim** [comment]
As a player I also experienced this lock up. For the job upgrade to assassin, there is this 10-question exam and after that is an additional quest to attack monsters, but soon after the exam, there supposed to be a portal to the next area. But theres no portal, not even a way out to restart the exam or out of the map (except restarting). I just repeatedly talk to the npc waiting room whenever i pass infront of the npc, and to the "anonymous" examiner npc. Nothing is happening after that. Im playing on renewal.

**Comment by aleos89** [comment]
I just ran through the Assassin Job Change Quest using e92fc96. I got to the part you talked about but had no issues walking through the warp portal that's down the stairs a bit.
It does warp you back to the entrance of that room because of this bit of code:
```
in_moc_16,182,169,0	script	Maze Assistant	45,1,1,{
OnTouch_:
	if (ASSIN_Q == 5 || ASSIN_Q == 6) {
		warp "in_moc_16",181,183;
		set ASSIN_Q,ASSIN_Q+1;
		changequest 8005,8006;
	}
```
I did get a `changequest` warning since it tried to give me quest 8006 twice. Other than that I got through just fine.
